### PR TITLE
Dragon's Descent : Void Dwellers

### PR DESCRIPTION
**Pull request by Breadbox2k19**
Patched all the stuff.
Dragon breathes are really iffy tho, sometimes they work sometimes they don't, it might be better if they simply used the vanilla shoot verb.
The armour penetration of their breath attack is another balance issue, I want them to be able to damage one another via breathe attack, but it seems uncertain at times.

Also, buffed archotech centipedes a bit to keep up with the power creep.
Also, please put a BIG note on the CE page indicating that just because the author says it is CE incompatible, doesn't mean it is actually CE incompatible, a lot of the mods are patched on our side and do check the compatibility list first!
## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [x] (For compatibility patches) ...with and without patched mod loaded
- [x] Playtested a colony (specify how long)

### CODE REVIEW

File: Patches/RWY Dragon's Descent Void Dwellers/Races_Animal_SpaceDragon_Base.xml [hunk 1]
```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<!-- I took 3 months off of RimWorld and I've forgotten how to do patches. Great. -->
```
Comment (MadaraUchiha) [inline Patches/RWY Dragon's Descent Void Dwellers/Races_Animal_SpaceDragon_Base.xml:3]: 😢

File: Patches/RWY Dragon's Descent Void Dwellers/Races_Animal_SpaceDragon_Base.xml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[@Name="VoidDragonRaceBase"]/race/manhunterOnTameFailChance</xpath>
+					<value>
+						<manhunterOnTameFailChance>0.6</manhunterOnTameFailChance>
+					</value>
+				</li>
+				<!-- ====== verbs (Because XPath dumb) ======== -->
+				<!--<li Class="PatchOperationAdd">
```
Comment (MadaraUchiha) [inline Patches/RWY Dragon's Descent Void Dwellers/Races_Animal_SpaceDragon_Base.xml:83]: Remove commented out code

**Review by MadaraUchiha — APPROVED** [review]
P-LGTM, please go over the files and format them to eliminate mixed tabs/spaces. Pick one indent style and stick with it.

File: Patches/RWY Dragon's Descent Void Dwellers/Races_Animal_SpaceDragon_Base.xml (hunk 1 above)
Comment (Breadbox2k19) [reply Patches/RWY Dragon's Descent Void Dwellers/Races_Animal_SpaceDragon_Base.xml:3]: That's not me, 😅 it's aluminium alman which made the original dragon's descent patch, I never take breaks.

**Review by N7Huntsman — APPROVED** [review]
All ranged attacks and stats seem to work fine. Some of the ranged attacks could probably be tweaked so they spread across a wider area, but they're fine as is.

**Pull request merged**